snapshot/snapshot.go

```diff
@@ -310,6 +312,14 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 					return nil, err
 				}
 			}
+			_, isDataLater := base.Labels[label.NydusDataLayer]
+			if isDataLater {
+				err = o.fs.PrepareBlobLayer(ctx, s, base.Labels)
```

Comment:
This is a highly time consuming operation, do we need to handle concurrency in `Prepare`?

Reply:
Will be supported soon